docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3.7"
+services:
+  expose:
+    image: lukepolo/expose-server:latest
```

Comment:
This will need to changed to your orgs / repo in docker hub. I left this for testing purposes.